unlock-app/src/components/creator/FatalError.js

```diff
@@ -84,7 +84,7 @@ export const WrongNetwork = ({ currentNetwork, requiredNetwork }) => (
     illustration="/static/images/illustrations/network.svg"
   >
     <p>
-      You’re currently on the
+      You’re currently on the 
```

Comment:
Worth seeing if your editor can automatically trim whitespace from the end of lines. It's not a big deal but it'd be nice to keep the source clean.

Reply:
Meh. this is weird. I thought prettify would take care of that?